sys/net/gnrc/sock/udp/gnrc_sock_udp.c

```diff
@@ -215,8 +215,9 @@ ssize_t sock_udp_recv(sock_udp_t *sock, void *data, size_t max_len,
         return -EPROTO;
     }
     memcpy(data, pkt->data, pkt->size);
+    int pkt_size = (int)pkt->size;
```

Comment:
Dito.

Reply:
> Please format the commit message properly. Please have a look at our Contribution Guidelines for that. Here's the TL;DR:

Ups i totally missed that part. sorry i will fix it in the next update :)